src/components/TreeMenu/parts/Details/Details.tsx

```diff
@@ -12,12 +12,17 @@ export const Details: FunctionComponent<DetailsProps> = props => {
 
     let content = children;
     if (elementsList) {
-        content = elementsList.map(element => <a className={cx('link')} href={element.path}>{element.title}</a>)
+        content = elementsList.map(element => <a className={cx('link')} href={element.path}><li>{element.title}</li></a>)
```

Comment:
Use icon `<BsDot />` with `margin-right` instead` <li> `tag. Because default browser margin is big and https://caniuse.com/#feat=css-marker-pseudo (chrome not support `::marker` pseudo-element).

Reply:
About the current file path. You can use `window.location` and React `useParams` hook to get the file path.

At TreeMenu component we can set prop open `<Details open={true}>` if <detаils> is part of file path.